library/psa_crypto_slot_management.c

```diff
@@ -315,6 +316,7 @@ static psa_status_t psa_load_builtin_key_into_slot(psa_key_slot_t *slot)
     /* Copy actual key length and core attributes into the slot on success */
     slot->key.bytes = key_buffer_length;
     slot->attr = attributes.core;
+    slot->status = PSA_SLOT_OCCUPIED;
```

Comment:
Same here, the slot shouldn't be marked as occupied when the function returns with an error code.

Reply:
I may be mistaken here but it seems to me that this assignment only occurs when `PSA_SUCCESS` will be returned, due to the check at line 312.

Reply:
Ah yes, sorry I commented on the wrong line.